src/main/java/cn/nukkit/form/element/ElementButton.java

```diff
@@ -0,0 +1,21 @@
+package cn.nukkit.form.element;
+
+public class ElementButton {
```

Comment:
@Creeperface01 Is this normal... Or should it `extends Element`... or isn't?

Reply:
It shouldnt, because button can be added only to simple Windows, not custom